### PR TITLE
fix linter style issues and remove spin_thread=False statement

### PR DESCRIPTION
**Pull request by kurshakuz**
Fix linter style issues and remove spin_thread=False statement #39

Signed-off-by: Shyngyskhan Abilkassov abilkasov@gmail.com

### CODE REVIEW

File: turtle_tf2_py/turtle_tf2_py/turtle_tf2_listener.py [hunk 1]
```diff
@@ -43,6 +43,7 @@ def __init__(self):
             self.get_logger().info('service not available, waiting again...')
 
         # Initialize request with turtle name and coordinates
+        # Note that x, y and theta are defined as floats in turtlesim/srv/Spawn
         request = Spawn.Request()
         request.name = 'turtle2'
         request.x = float(4)
```
Comment (ahcorde) [inline turtle_tf2_py/turtle_tf2_py/turtle_tf2_listener.py:49]: You can do this: 
```python
request.x = float(4)
# or
request.x = 4.0
```

Comment (kurshakuz) [reply turtle_tf2_py/turtle_tf2_py/turtle_tf2_listener.py:49]: Do you mean that I should replace typecasting with adding a floating point on these 3 values?

Comment (ahcorde) [reply turtle_tf2_py/turtle_tf2_py/turtle_tf2_listener.py:49]: not necessarily, I was just adding a comment, it more a Python related thing, maybe it does not make sense to do nothing

**Review by audrow — APPROVED** [review]
Looks good to me!

**Review by ahcorde — APPROVED** [review]

**Pull request merged**